pkg/controller/authentication/authentication_controller.go

```diff
@@ -243,14 +239,14 @@ func (r *ReconcileAuthentication) Reconcile(ctx context.Context, request reconci
 	// Validate that the tier watch is ready before querying the tier to ensure we utilize the cache.
 	if !r.tierWatchReady.IsReady() {
 		r.status.SetDegraded(oprv1.ResourceNotReady, "Waiting for Tier watch to be established", nil, reqLogger)
-		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
+		return reconcile.Result{RequeueAfter: utils.StandardRetry}, nil
```

Comment:
This falls into the category for me of "if 10 seconds is good enough, so is 30"

Or alternatively, "If 10 seconds is better than 30, then why not do 1s?"

Reply:
This is potentially adding 20s to our rollout. I would say it is a trade off, 1s would be spamming the logs and requests and 30s is slow to respond. I realize that we'll eventually become ready but this is a place we won't be triggered when the state changes so this isn't a backstop like (I think) it is in most places.

Reply:
I don't disagree that 30 seconds is a longer time, and 1s would produce more logs (unless we downgraded them). But how do we make the call that 10s is correct and not 12, or 15, or 20? It's just an arbitrary number and if someone had coded this up as 30s originally we wouldn't be having this conversation. To me, that means there's no logical reason to make this 10s instead of 30s.

Sure, you could use the same logic to say "Why not make it an hour?" - but I think that's taking the argument to absurdity, when we would otherwise be discussing something in the same order of magnitude.

I also disagree that adding 20s of time here (in some but not  all, deployments) is fundamentally something we should care about or is worth adding a second constant for.

Reply:
> I think this will slow down our rollout because I don't think the tierWatch becoming ready triggers a reconcile

taking a step back, why doesn't the tier watch becoming ready trigger a reconcile? I agree that the watch establishment itself wouldn't do this, but I really would have expected that establishing the watch would have the consequence of filling the cache with tier events, which in turn trigger reconciles.

Reply:
And just to really drive home the 20s difference thing.... it takes between 5-8 minutes to install Elasticsearch in our out-of-the-box install right now.

Reply:
That is an excellent point that the tierWatch being established should result in tier events triggering reconcile.
I don't know that just because a part of our rollout already takes too long is a valid argument that we should be ok to make it longer. But because of what you've pointed out about the tierWatch means there shouldn't be any need for the RequeueAfter we don't need to worry about that here.